code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -217,3 +217,17 @@
 		B.deity_name = "Success"
 		B.icon_state = "greentext"
 		B.item_state = "greentext"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box
+	name = "A box. An unassuming box of WAR!"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box/New()
+	..()
+	new /obj/item/weapon/storage/backpack/duffel/syndie/ammo/assault(src)
```

Comment:
What about a larger box? We have those but I forget what fits inside them. I just use them for more item storage